packages/connector/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rarimo/rarime-connector",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Does this change extend the public API/UI of the snap and the connector in any way? If no, this should a patch version according to semver. So the version should be `0.2.1`